modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesContextInformationManager.java

```diff
@@ -41,21 +39,16 @@
     private static final Log log = LogFactory.getLog(AndesContextInformationManager.class);
 
     //keep listeners that should be triggered when constructs are updated
```

Comment:
These needs to be a java doc style comments. (for each variable, lets keep our practice of documenting each variable alive!)